code/game/objects/items/grenades/clusterbuster.dm

```diff
@@ -55,13 +55,13 @@
 		max_spawned = base.max_spawned
 	icon_state = "[base_state]_active"
 	active = TRUE
-	var/steps = rand(1,4)
-	for(var/i in 1 to steps)
-		step_away(src,loc)
-	addtimer(CALLBACK(src, .proc/detonate), rand(15,60))
+	var/steps = rand(1, 4)
+	for(var/step in 1 to steps)
+		step_away(src, loc)
+	addtimer(CALLBACK(src, .proc/detonate), rand(15, 60))
```

Comment:
These 15 and 60s come up in a lot of places. Can you move these to a define of `RANDOM_DETONATE_MIN_TIME` and `RANDOM_DETONATE_MAX_TIME`, then use the `SECONDS` defines, and replace all of them?